buildSprite.js

```diff
@@ -0,0 +1,184 @@
+const path = require('path');
+const fs = require('fs');
+const chunk = require('lodash.chunk');
+const sharp = require('sharp');
+const imagemin = require('imagemin');
+const imageminPngquant = require('imagemin-pngquant');
+const { getCodes } = require('country-list');
```

Comment:
These codes don't all match with the country codes in our DB, and there's at least one missing. For instance, the country code for Palestine (Gaza) in our DB is `gz` but in the country-list package it's `ps`. The same is true for South Sudan; `qs` in our DB but `ss` in that package. Also, we have Kosovo in our DB as `xk`, but that's not in that package at all.

On a related note it also looks like the svg files in here for Palestine and South Sudan use `ps` and `ss` instead of `gz` and `qs`, though fortunately it does appear to have the Kosovo flag under `xk`.

Reply:
Couldn't we just loop through all the svg files in the directory instead of using the country-list package at all?

Reply:
> Couldn't we just loop through all the svg files in the directory instead of using the country-list package at all?

I switched to this package as a way to avoid hard-code the countries array in KvFlag.vue so we stay in sync. 
https://github.com/kiva/ui/pull/2004/files#diff-3d0e5ea7e4155a8da099ac6ee50f465dR24

Reply:
> These codes don't all match with the country codes in our DB, and there's at least one missing. For instance, the country code for Palestine (Gaza) in our DB is `gz` but in the country-list package it's `ps`. The same is true for South Sudan; `qs` in our DB but `ss` in that package. Also, we have Kosovo in our DB as `xk`, but that's not in that package at all.
> 
> On a related note it also looks like the svg files in here for Palestine and South Sudan use `ps` and `ss` instead of `gz` and `qs`, though fortunately it does appear to have the Kosovo flag under `xk`.

@emuvente Thanks for checking that out! Can you possibly get me a list of country codes used in our DB so I can make a .story and see what's messed up?

Reply:
I imagine it's going to be easier to handle the different country codes in this repo rather than tweak our DB.

Reply:
Sure, I'll slack that to you.

Reply:
> I switched to this package as a way to avoid hard-code the countries array in KvFlag.vue so we stay in sync.

I see the use for it when getting the country names, but it's not needed here for just generating the sprite sheet.

Reply:
Following up on this, using the country codes from our DB, I added a json file with country code mappings to this repo. This repo now uses the list to build the sprite, and exports some functions to allow our UI repo to access the same list. Doing so allows KvFlag to know the height of this sprite and set the background position accordingly.